cmd/livepeer/livepeer.go

```diff
@@ -311,6 +318,21 @@ func main() {
 		n.EthServices["UnbondingService"] = eventservices.NewUnbondingService(n.Eth, dbh)
 
 		if *orchestrator {
+
+			// Set price per pixel base info
+			if *pixelsPerUnit == 0 {
+				// Can't divide by 0
+				glog.Fatalln("The amount of pixels per unit must be greater than 0")
```

Comment:
Wouldn't this always result in the underlying ? I guess it could be a negative integer as well though, I just didn't ever see a user entering a negative number there that's why I omitted the provided value. It did cross my mind. 

> pixelsPerUnit must be greater than 0, but 0 provided

Reply:
> I guess it could be a negative integer as well though

Right and in fact we should probably ensure that `*pricePerUnit` and `*pixelsPerUnit` are not negative integers (we're just checking that they are not 0 at the moment).

While users typically wouldn't enter negative numbers, we should still make sure that our code is able to handle atypical input.

Reply:
Entirely correct and thus changed the checks to check for `<= 0` and will add tests for negative ints as well.